test/unit/ngsiv2/contextRequests/timeInstantMeasures.json

```diff
@@ -0,0 +1,16 @@
+{
+  "humidity":{
+    "type": "percent",
+    "value": "111222",
+    "metadata":{
+      "TimeInstant":{
```

Comment:
This (and the other TimeInstance usage in L12 below) could be problematic at CB. Is `20200222T222222` a valid ISO8601 format?

I'd suggest to test this literrarly (e.g. a crafted curl using this very same JSON file) in a real Orion instance.

Reply:
You are right. Orion anwers:

```
{"error":"BadRequest","description":"date has invalid format"}
```

Reply:
Now we have the result of the test (thx!) next question is:

Is `20200222T222222` a valid ISO8601 (but not supported by Orion)? Or an invalid ISO8601 string at all?

Reply:
I have just tried https://www.npmjs.com/package/iso8601-validator and it recognizes `20200222T222222` as a valid ISO8601 string.

I have also checked this in some other resources and it seems that is format is what they call ``basic format`:
- From https://en.wikipedia.org/wiki/ISO_8601:
> For example, "5 April 1981" may be represented as either "1981-04-05"[12] in the extended format or "19810405" in the basic format.

> So a time might appear as either "134730" in the basic format or "13:47:30" in the extended format.

Reply:
My conclusion: 20200222T222222 is a valid ISO8601 string using basic format, which seems to be not supported by Orion.

Reply:
Your analysis seems to be correct. The following documentation describes the ISO8601 support in Orion for DateType: https://fiware-orion.readthedocs.io/en/master/user/ngsiv2_implementation_notes/index.html#datetime-support and basic type part of that support.

I have created an issue about the new Orion functionality: https://github.com/telefonicaid/fiware-orion/issues/3085. In the hope that can come sooner than latter, let's keep the PR as it is now (i.e. mock working as if Orion would have implemented https://github.com/telefonicaid/fiware-orion/issues/3085) and take this into account in the future (e.g. to add a note to IOTAgent node lib documentation explaining that data compression plugin is not going to work with v2 configuration).

NTC (with regards the present PR), from my point of view.